harmony/harmony.py

```diff
@@ -209,18 +211,49 @@ def _temporal_subset_params(self, request: Request) -> list:
         else:
             return []
 
-    def submit(self, request: Request) -> Optional[dict]:
+    def submit(self, request: Request) -> Optional[str]:
         """Submits a request to Harmony and returns the Harmony job details."""
         if not request.is_valid():
             msgs = ', '.join(request.error_messages())
             raise Exception(f"Cannot submit an invalid request: [{msgs}]")
 
-        job = None
+        job_id = None
         session = self._session()
-        response = session.get(self._url(request), params=self._params(request)).result()
+        response = session.get(self._submit_url(request), params=self._params(request)).result()
         if response.ok:
-            job = response.json()
+            job_id = (response.json())['jobID']
         else:
             response.raise_for_status()
 
-        return job
+        return job_id
+
+    def status(self, job_id: str, progress_only=False) -> Union[str, dict]:
+        """Returns job metadata from Harmony.
+
+        If ``progress_only`` is True, return the job's percent complete. If ``progress_only`` is
+        False, return dict containing job metadata.
+
+        Args:
+            job_id: UUID string for the job you wish to interrogate.
+            progress_only: An option to change the output format.
+
+        Returns:
+            Either the percent complete or a dict of metadata.
+
+        :raises Exception: This can happen if an invalid job_id is provided or Harmony services
+        can't be reached.
+        """
+        session = self._session()
+        response = session.get(self._status_url(job_id)).result()
+        if response.ok:
+            if progress_only:
+                return (response.json())['progress']
+            else:
+                fields = [
```

Comment:
We might want to python-ize the names (camel -> snake case)

Reply:
Sure - as we've been making harmony-py it's sometimes hard to know how much of Harmony should be represented as-is vs. re-interpreting output on behalf of the user.